src/commands/user/lookup.ts

```diff
@@ -1,7 +1,72 @@
-import {Message} from 'discord.js'
+import fetch from 'node-fetch';
+import { Message, MessageEmbed, MessageSelectMenu } from 'discord.js'
+import { responseHandler } from '../../utils/responseHandler.js'
+
+export async function lookUp(message: Message, _arguments: string[]) {
+
+	let apiURL;
```

Comment:
Infer type `IapiEndpoints`

Reply:
wdym `IapiEndpoints`  ?     ? ?

Reply:
Create a type that holds API endpoints...

```ts

type IapiEndpoints = "url1" | "url2" | "url3"...
```

Reply:
> Create a type that holds API endpoints...
> 
> ```ts
> type IapiEndpoints = "url1" | "url2" | "url3"...
> ```

there is id behind them
![image](https://user-images.githubusercontent.com/60780448/147252557-4a1ac1ca-9ce1-4b4c-9a7b-7cd6abdef143.png)

![image](https://user-images.githubusercontent.com/60780448/147252504-a6666c8d-7975-4b02-b76e-2333d34ff864.png)